drivers/SmartThings/matter-lock/src/new-matter-lock/init.lua

```diff
@@ -1065,6 +1068,15 @@ end
 -----------------------------
 -- Set Credential Response --
 -----------------------------
+local RESPONSE_STATUS_MAP = {
+  [DoorLock.types.DlStatus.FAILURE] = "failure",
+  [DoorLock.types.DlStatus.DUPLICATE] = "duplicate",
+  [DoorLock.types.DlStatus.OCCUPIED] = "occupied",
+  [DoorLock.types.DlStatus.INVALID_FIELD] = "invalidCommand",
+  [DoorLock.types.DlStatus.RESOURCE_EXHAUSTED] = "resourceExhausted",
+  [DoorLock.types.DlStatus.NOT_FOUND] = "failure"
```

Comment:
Should this be "notFound" instead of "failure"?

Reply:
I guess not because "notFound" is not a valid value for the `statusCode` property in the capability

Reply:
Yes, there's no `notFound` value for capability